.gitignore

```diff
@@ -1,2 +1,5 @@
 node_modules
-dist/
\ No newline at end of file
+dist/
+package-lock.json
+yarn.lock
```

Comment:
Is there any reason we do not want to commit package-lock.json?

Reply:
this can be reverted to include package-lock

Reply:
Yep. we do not exclude them in other demos.